target/linux/ramips/image/mt76x8.mk

```diff
@@ -29,6 +29,16 @@ define Build/ravpower-wd009-factory
 endef
 
 
+define Device/360_p2
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := 360
+  DEVICE_MODEL := P2
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-usb-ledtrig-usbport \
+	kmod-mt76x2 
+  SUPPORTED_DEVICES += p2
```

Comment:
This can be dropped.

Reply:
So what's wrong here? I am confusing about it.

Reply:
Why do you add "p2" to the supported devices?

Reply:
Has been removed. I saw many devices have this statement. Does it is not used to verify device model when apply sysupgrade?

Reply:
Yes, but do you know a device that uses "p2" as its compatible?

Reply:
Finally I understand. The device define of ramips in makefile does not contain vendor name before v21.02. It will cause compatibility problem if not add SUPPORTED_DEVICES, but new support devices don't have this historical problem.
The commit has been force pushed and I think all issues have been solved.